src/main/java/seedu/address/model/person/Role.java

```diff
@@ -45,6 +47,9 @@ public static boolean isValidRole(String test) {
         }
     }
 
+    public RoleType getRoleType() {
+        return role;
+    }
```

Comment:
In retrospect, we should have made `Role` an enum instead of a class containing an enum. (**An enum is a class in Java.**)

Also this violates tell-don't-ask. The proper way is of course `this.equals(Role("PROFESSOR"))`, but we refactor `Role` first.

Reply:
So what changes should I make for this right now?

Reply:
Essentially refactor the `Role` class to be an enum itself, if you can do that quickly.
(should pretty much not change its external behavior, I think.)